examples/migrate_forseti/tutorial.md

```diff
@@ -0,0 +1,362 @@
+questions about this process, please contact us by
+e-mail at discuss@forsetisecurity.org or on
+[Slack](https://forsetisecurity.slack.com/join/shared_invite/enQtNDIyMzg4Nzg1NjcxLTM1NTUzZmM2ODVmNzE5MWEwYzAwNjUxMjVkZjhmYWZiOGZjMjY3ZjllNDlkYjk1OGU4MTVhZGM4NzgyZjZhNTE).
+
+## Prerequisites
+
+Before you begin the migration process, you will need:
+
+- A Forseti deployment of at least v2.18.0.  You must follow the
+  [upgrade guide](https://forsetisecurity.org/docs/latest/setup/upgrade.html) to be at least v2.18.0.
+- A version of the
+  [Terraform command-line interface](https://www.terraform.io/downloads.html)
+  in the 0.12 series.
+- The domain and ID of the Google Cloud Platform (GCP) organization in
+  which Forseti is deployed.
+- The ID of the GCP project in which Forseti is deployed.
+- The suffix appended to the names of the Forseti resources; this is
+  likely a string of seven characters like a1b2c3d.
+- A service account for the organization with the
+  [roles required by the Terraform module](https://registry.terraform.io/modules/terraform-google-modules/forseti/google/5.0.0#iam-roles).
+- A
+  [JSON key file](https://cloud.google.com/iam/docs/creating-managing-service-account-keys#creating_service_account_keys)
+  for the service account.
+- If you are an Org Admin in the organization in which you deploying Forseti, a separate Service Account and Key are recommended,
+  but not required.
+- **Strongly recommended out of an overabundance of caution:** A backup of your current state.
+  - In the Forseti Server's GCS Bucket
+    - Scanner rules
+    - Server config file
+    - Scanner Violations
+    - Inventory summary
+  - [CloudSQL database](https://cloud.google.com/sql/docs/mysql/backup-recovery/backups)
+
+If you deployed Forseti in a shared VPC then you will also need:
+
+- The ID of the GCP project in which the shared VPC is hosted.
+- The ID of the shared VPC network in which Forseti is deployed.
+- The ID of the subnetwork from the shared VPC network in which Forseti is deployed.
+
+
+## Configure gcloud
+In the Cloud Shell, configure `gcloud` to use the GCP in which Forseti is deployed.
+```bash
+gcloud config set project PROJECT_ID
+```
+
+## Configuring Terraform
+Terraform can assume the identity of a service account through a
+strategy called
+[Application Default Credentials](https://cloud.google.com/docs/authentication/production#providing_credentials_to_your_application)
+when provisioning resources. To enable this approach, set the
+appropriate environment variable to the path of the service account JSON
+key file:
+
+```sh
+export GOOGLE_APPLICATION_CREDENTIALS="PATH_TO_JSON_KEY_FILE"
+```
+As stated in the pre-requisites, if you have Org Admin privilges, you do not need to complete this step.
+
+## Configure Forseti
+To install Forseti, you will need to update a few settings in the <walkthrough-editor-open-file filePath="terraform-google-forseti/examples/migrate_forseti/main.tf">main.tf</walkthrough-editor-open-file>.
```

Comment:
@hshin-g @gkowalski-google Ken is asking people to update the variables in the `main.tf` here (instead of tfvars/variables.tf).  Should we be consistent?

Reply:
If we are expecting users to use this migration as their Terraform configuration going forward, then I guess so. In the end we provide steps for backing this up and using for future use, so it seems like yes these updates should actually be mnade in tfvars/vars.tf.

Reply:
Customers should not be editing `variables.tf` as they should be modifying their configuration of deployment. Hence, `main.tf` need to be updated.